Cars/CarsRepository/Interface/ICarRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CarsData.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CarsRepository.Interface
+{
+    public interface ICarRepository
+    {
+        IEnumerable<Car> GetAllCars();
+
+        IEnumerable<Car> GetFavoriteCars { get; }
+
+        IEnumerable<Car> GetCarsByCategory(int categoryId);
+        Car GetObjectCar(int CarId);
+
+        void SaveCar(Car Cars);
```

Comment:
Its OK, but I suggess you to return smth after such moves like delete, add and so on. Return bool value, for example if delete sucess return true, else return false.